sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/_tests_/e2e/confirmation-of-enrollment.institutions.getCOESummary.e2e-spec.ts

```diff
@@ -130,6 +130,47 @@ describe("ConfirmationOfEnrollmentInstitutionsController(e2e)-getCOESummary", ()
       });
   });
 
+  it("Should get the count of COE current summary as zero when there is 1 COE that doesn't have estimated awards.", async () => {
```

Comment:
Please add the E2E test for [confirmation-of-enrollment.institutions.confirmEnrollment.e2e-spec.ts](https://github.com/bcgov/SIMS/blob/c36dad4423fe557a222ec39b87b09edd7c7a0520/sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/_tests_/e2e/confirmation-of-enrollment.institutions.confirmEnrollment.e2e-spec.ts#L30)

AND 

[confirmation-of-enrollment.institutions.getApplicationForCOE.e2e-spec](https://github.com/bcgov/SIMS/blob/main/sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/_tests_/e2e/confirmation-of-enrollment.institutions.getApplicationForCOE.e2e-spec.ts)